qemu/tests/qemu_guest_agent.py

```diff
@@ -1356,8 +1433,6 @@ def setup(self, test, params, env):
                                                      session, params, self.vm)
             self.gagent_install_cmd = params.get("gagent_install_cmd"
                                                  ) % qemu_ga_pkg_path
-            self.gagent_uninstall_cmd = params.get("gagent_uninstall_cmd"
-                                                   ) % qemu_ga_pkg_path
```

Comment:
Because self.gagent_uninstall_cmd is defined in __init()__ before,no need define here.
self.gagent_install_cmd defined here as need to replace the placeholder

Reply:
@xiagao Well, I would request to handle this separately with another patch.

Reply:
ok, I sent a new patch for it just now, pls have a check.